api-report/purchases-js.api.md

```diff
@@ -170,6 +170,7 @@ export type PeriodType = "normal" | "intro" | "trial";
 export interface Price {
     readonly amount: number;
     readonly currency: string;
+    readonly formattedPrice: string;

```

Comment:
Also, note that since this will be a precomputed-property, it won't respond to locale changes... But I think it's ok. Lmk if you think otherwise.